rest_framework_docs/api_endpoint.py

```
@@ -59,6 +65,75 @@ def __get_serializer_fields__(self):
 
         return fields
 
+    def __get_filters__(self, filter_backend):
+        filter_data =  {"backend_name": filter_backend.__name__,
+                        "filter_fields": [],
+                        "search_param": None,
+                        "ordering": {}
+                        }
+
+        filter_class = None
+        get_filter_class = getattr(filter_backend, "get_filter_class", None)
+        if callable(get_filter_class):
+            try:
+                try:
+                    qs = self.callback.cls().get_queryset()
+                except:
```

bare except. can you use `except Exception` here?